### DISCUSSION TITLE
login with

### DISCUSSION THREAD

**Discussion by Turkidev** [pelican-dev/panel#308] [Ideas]
is there a way to add login with "Google", "Discord", "Github"

**Comment by Boy132** [comment]
Not yet but it is planned.

**Reply by ItsNeil17** [reply]
It's quite easy using the socialite plugin: https://v2.filamentphp.com/plugins/socialite

SSO would be easy, but adding configurable options for the user to configure themselves would be the hard part.

**Reply by Boy132** [reply]
> It's quite easy using the socialite plugin: https://v2.filamentphp.com/plugins/socialite
> 
> SSO would be easy, but adding configurable options for the user to configure themselves would be the hard part.

Yes, the plan is to use socialite.

**Comment by lancepioch** [comment]
Planned already, thanks.
